src/views/Layout.vue

```diff
@@ -1,38 +1,78 @@
 <template>
-  <div class="container">
-    <h1>Vue.js layout</h1>
-    <!-- Render the files data with a resuable component (a list or card up to you!) -->
-    <pre>// TODO: replace this block 👇 with a resusable component that renders elements from the files array 
-      <code>files:
-      {{files}}</code>
-    </pre>
+  <div class="main-container">
+    <h1>Pictures of things I like...</h1>
+    <Kitten
+      v-if="files.length"
+      :nameOfFile="files[index]"
+      :next="next"
+      :previous="previous"
+    />
+    <div>
+      <p>
+        Search for a picture with a tag.
+        <br />
+        <em> eg. Kitten </em>
+      </p>
+      <div>
+        <!-- <div v-if="state === 'edit'"> -->
+        <input v-model="newTag" placeholder="Search" />
+        <p>{{ tagsCount }}</p>
+        <input type="Submit" />
+      </div>
+      <TagsFilter />
+    </div>
   </div>
 </template>
+
 <style lang="scss" scoped>
-.container {
-  max-width: 1024px;
-  margin: 0 auto;
+h1 {
+  font-family: $heading-font-family;
+  text-align: center;
 }
-
-pre,
-code {
-  overflow-x: auto;
-  white-space: pre-wrap;
-  white-space: pre-wrap;
-  line-break: anywhere;
+.main-container {
+  margin: 0 auto;
 }
 </style>
+
 <script>
 // @ is an alias to /src
 import api from "@/api";
+import Kitten from "./Kitten.vue";
+import TagsFilter from "./TagsFilter.vue";
```

Comment:
@Lilish-Warrior did you forget to `git add TagsFilter.vue` ? Can't find it in the repo.

Reply:
Hi! 
I kept on working on the project after I made the PR, and that is another component I want to work on, but I still haven't finished it.
I didn't realised that component went into the PR!! Sorry it's not done yet! 😅

Reply:
@kakauandme Just added it but it's an empty file atm..

Reply:
Ok, no worries. I'll comment it out for now.